javascript/chronometer.js

```diff
@@ -1,26 +1,43 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime= 0;
+    this.intervalId= 0;
+
   }
   startClick(callback) {
-    // ... your code goes here
-  }
-  getMinutes() {
-    // ... your code goes here
-  }
-  getSeconds() {
-    // ... your code goes here
-  }
-  twoDigitsNumber() {
-    // ... your code goes here
-  }
+
+this.intervalId = setInterval(() => this.currentTime++;
+callback()
+}, 1000)
+
+}
+
+  getMinutes(currentTime) {
+    return (Math.floor(this.currentTime/60))
+  }
+  getSeconds(currentTime) {
+    var minutes= this.getMinutes(currentTime)
+    
+    return (Math.floor(this.currentTime - (minutes * 60)))
+  }
+  twoDigitsNumber(currentTime) {
+    
+     currentTime =  "0" + currentTime       
```

Comment:
You should not update the currentTime with a string. Doing this will cause the counter you have in the setInterval to break and start concatenating strings instead of adding 1 to the counter. Instead you just want to return the string for when the function is called later.